### DISCUSSION TITLE
October does not open

### DISCUSSION THREAD

**Discussion by jHettema** [marcus-crane/october#99] [Q&A] [answered]
Hi everyone, 

Somehow October does not open anymore. Any ideas what might be the issue?

I've tried reinstalling and rebooting, but no luck. Im on a Mac, Sonoma 14.0

Thanks!

**Comment by marcus-crane** [comment]
Hey @jHettema,

Hmm, this sounds quite similar to this issue: https://github.com/marcus-crane/october/issues/98

If you navigate to `/Users/<username>/Library/Application Support/October`, you should see a file called `config.json` which contains October's settings such as your Readwise token and sync preferences.

You can get there by opening Finder and then navigating to `Go -> Go to Folder`.

Try renaming it to say; `config.old.json` and see if that helps October to start up.

If that does the trick, I'll prioritise looking into the issue as I've only had one report of it.

Just to confirm, this is using the latest version of October (v1.6.0)?

You can confirm the version by opening Finder and then navigating to `Applications -> October.app` and looking the Version field as shown just below

![CleanShot 2023-11-16 at 14 49 07](https://github.com/marcus-crane/october/assets/14816406/8adb6899-6cba-43fd-b137-8d9b42f842bc)

**Reply by marcus-crane** [reply]
Just confirming that I've managed to replicate this issue. It's a bug where it can potentially try to save the settings file twice in quick succession in an unsafe manner which causes the file to get corrupted. I'll push a new version shortly that should fix the issue.

**Reply by marcus-crane** [reply, marked as answer]
Hey there,

I've just release a pre-release version of October: https://github.com/marcus-crane/october/releases/tag/v1.7.0-beta11

Updating to use it should fix your config file when you next try to start up the new version of October

I still need to run it across the various platforms just to confirm it all works as there are some other changes in there as well but in the meantime, you can give it a go

**Reply by jHettema** [reply]
Thank you Marcus, this version opens again like normal! if anything comes up I will let you know.

Very kind of you to fix this so quickly, and thank you for all the great work.

**Reply by marcus-crane** [reply]
Hey there,

No worries, thanks for reporting this issue. In future, feel free to open a bug report under the Issues tab or any feature requests that might come up.

I still need to do some testing on various platforms but I'll roll out the fix officially in a bit. It won't contain anything different than the pre-release version you've got though.
